src/Shared/Microsoft.AspNetCore.Razor.Utilities.Shared/StringExtensions.cs

```diff
@@ -552,6 +552,31 @@ public static bool EndsWith(this string text, char value)
         return text.EndsWith(value);
 #else
         return text.Length > 0 && text[^1] == value;
+#endif
+    }
+
+    public delegate void SpanAction<T, in TArg>(Span<T> span, TArg arg);
+
+    public unsafe static string CreateString<TState>(int length, TState state, SpanAction<char, TState> action)
```

Comment:
Other methods in the file have triple slash docs on them. Would it be possible to add one for `CreateString`?

Reply:
Sure thing! I missed adding an XML doc comment to this one. Thanks!